Tutorials/Day_3/Tuto_3.3_Discovering_and_using_published_posterior_samples.ipynb

```diff
@@ -132,20 +132,20 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "--2025-03-06 16:09:23--  https://dcc.ligo.org/LIGO-P1800370/public/GW150914_GWTC-1.hdf5\n",
-      "131.215.125.133cc.ligo.org (dcc.ligo.org)… \n",
-      "connecté. à dcc.ligo.org (dcc.ligo.org)|131.215.125.133|:443… \n",
-      "requête HTTP transmise, en attente de la réponse… 302 Found\n",
-      "Emplacement : https://dcc.ligo.org/public/0157/P1800370/005/GW150914_GWTC-1.hdf5 [suivant]\n",
-      "--2025-03-06 16:09:24--  https://dcc.ligo.org/public/0157/P1800370/005/GW150914_GWTC-1.hdf5\n",
-      "Réutilisation de la connexion existante à dcc.ligo.org:443.\n",
-      "200 OKe HTTP transmise, en attente de la réponse… \n",
-      "Taille : 7026464 (6,7M)\n",
-      "Sauvegarde en : « GW150914_GWTC-1.hdf5 »\n",
+      "--2025-04-21 16:51:18--  https://dcc.ligo.org/LIGO-P1800370/public/GW150914_GWTC-1.hdf5\n",
+      "Resolving dcc.ligo.org (dcc.ligo.org)... 131.215.125.133\n",
+      "Connecting to dcc.ligo.org (dcc.ligo.org)|131.215.125.133|:443... connected.\n",
+      "HTTP request sent, awaiting response... 302 Found\n",
+      "Location: https://dcc.ligo.org/public/0157/P1800370/005/GW150914_GWTC-1.hdf5 [following]\n",
+      "--2025-04-21 16:51:19--  https://dcc.ligo.org/public/0157/P1800370/005/GW150914_GWTC-1.hdf5\n",
+      "Reusing existing connection to dcc.ligo.org:443.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 7026464 (6.7M)\n",
+      "Saving to: ‘GW150914_GWTC-1.hdf5’\n",
       "\n",
-      "GW150914_GWTC-1.hdf 100%[===================>]   6,70M  5,18MB/s    ds 1,3s    \n",
+      "GW150914_GWTC-1.hdf 100%[===================>]   6.70M  18.2MB/s    in 0.4s    \n",
       "\n",
-      "2025-03-06 16:09:25 (5,18 MB/s) — « GW150914_GWTC-1.hdf5 » sauvegardé [7026464/7026464]\n",
+      "2025-04-21 16:51:19 (18.2 MB/s) - ‘GW150914_GWTC-1.hdf5’ saved [7026464/7026464]\n",
       "\n"
```

Comment:
Obviously, this is not related to the topic but this made me realized that the output is in french. So maybe we should use the `--no-verbose` option to reduce noise.

Reply:
See #59 and #60

Reply:
or should I revert these lines?

Reply:
Yep, don't bother. This will surely create a conflict in #60 but I will manage this.